web-app/src/pages/Profile/container.js

```diff
@@ -56,7 +60,20 @@ const ProfileContainer = Wrapped =>
             });
           });
         });
-      }
+      } 
+      if (
+        this.props.userDetails.employeeId !== null &&
+        this.state.contracts !== null
+      )
+        {   
```

Comment:
Tabbing seems to be a bit off.

Reply:
So what happens if the conditions fails, it just doesn't run `getContractsByEmployeeId` is that expected behaviour?